src/mods.cpp

```diff
@@ -80,12 +80,15 @@ void parseModContents(ModSpec &spec)
 	}
 }
 
-std::map<std::string, ModSpec> getModsInPath(std::string path, bool part_of_modpack)
+std::map<std::string, ModSpec> getModsInPath(std::string path, bool part_of_modpack, std::string worldpath)
```

Comment:
Sorry, I thought you were seeing a compile warning since you commented on this. Is that not the case?

Am I missing something? I don't see const being used anywhere else in the way you suggest....